### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by AlexForJava**

### CODE REVIEW

File: src/main/resources/init.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(255) NOT NULL,
+  age INT NOT NULL,
+  male TINYINT(1) NULL,
+  PRIMARY KEY (id));
+
+CREATE TABLE skills (
+  id INT NOT NULL AUTO_INCREMENT,
+  branch VARCHAR(255) NOT NULL,
+  level VARCHAR(255) NOT NULL,
```
Comment (Anderwerty) [inline src/main/resources/init.sql]: It is better to create a separate table for levels and use it.  
The same for branch

File: src/main/resources/task2/2.1.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+ALTER TABLE developers ADD COLUMN salary INT NOT NULL;
+UPDATE developers
```
Comment (Anderwerty) [inline src/main/resources/task2/2.1.sql]: Please separate each request with an empty line. 
And if a request is not too long you could write it into one line

`UPDATE developers SET salary = 1000 WHERE id = 1;`

File: src/main/resources/task3/3.5.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
+ALTER TABLE projects ADD COLUMN date DATE;
+SELECT *
```
Comment (Anderwerty) [inline src/main/resources/task3/3.5.sql]: The same

**Review by Anderwerty — CHANGES_REQUESTED** [review]

**Review by Anderwerty — CHANGES_REQUESTED** [review]
Where is java code? Task #3?

**Comment by AlexForJava** [comment]
Later

11 апреля 2019, 01:10:05, от "Andrii Shylin" <notifications@github.com>:

@Anderwerty requested changes on this pull request.

Where is java code? Task #3?

**Review by Anderwerty — CHANGES_REQUESTED** [review]
What about java code. Are you going to push something?

**Review by Anderwerty — APPROVED** [review]

**Pull request merged**